src/UserManagerSettings.ts

```diff
@@ -101,12 +108,12 @@ export class UserManagerSettingsStore extends OidcClientSettingsStore {
         const {
             popup_redirect_uri,
```

Comment:
You could simplify some of these further with
```suggestion
            popup_redirect_uri = args.redirect_uri,
```